### PR TITLE
Lindsay Cooper

### PR DESCRIPTION
**Pull request by Lindsaysaracooper**
Another test

### CODE REVIEW

File: 03LessonThree/in-class-project/daily-planet.html [hunk 1]
```diff
@@ -1,10 +1,36 @@
...
+            <a href="local.com">local</a>
+            <a href="www.economy">economy</a>
+        </nav>
+    </header>
+    <main>
+        <section>
+            <article>
+                <h2> I Love Lamp </h2>
+                <p> Student loves lamp </p>
+                <img src="http://s7d1.scene7.com/is/image/BedBathandBeyond/3614813588384p" width="1" height="250" </>
```
Comment (bapplejax) [inline 03LessonThree/in-class-project/daily-planet.html:23]: great nesting on this, but I would give a little more vertical space between elements and comment a bit more :)  also, try not to set the width and height of the image in the tag; best to do this with CSS. so far so good :)

File: 04LessonFour/in-class-project/index.html [hunk 2]
```diff
@@ -2,64 +2,71 @@
...
 
         <p>
-            The 6th Generation Intel &reg; Core &Trade; i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lightening fast speeds and peak performance through even the
+            The 6th Generation Intel&reg; Core&Trade; i7 processors deliver a new class of computing with a host of new features to power the next generation of desktops, laptops, and 2 in 1 PCs. Expect lightening fast speeds and peak performance through even the
             toughest of tasks and games.
         </p>
     </section>
 
-    <section>
+    <section id=ready>
```
Comment (bapplejax) [inline 04LessonFour/in-class-project/index.html]: be careful to add in quotes when giving an element's attribute a property (like id="ready" instead of id=ready)

File: 04LessonFour/in-class-project/index.html [hunk 3]
```diff
@@ -2,64 +2,71 @@
...
         </p>
+      </section>
 
-        <aside>
-            <!--Figure this out-->
-            <img src=< /aside>
+        <aside id="aside_computer">
 
-    </section>
+            <img src="img/processor.jpg" </>
```
Comment (bapplejax) [inline 04LessonFour/in-class-project/index.html]: be on the lookout for auto-complete stuff. not sure if you meant to, but there is an open and close tag at the end of this img ( </> ). also, it's a good idea to alt and title your images.

File: 04LessonFour/in-class-project/index.html [hunk 4]
```diff
@@ -2,64 +2,71 @@
...
+
+        </aside>
 
-    <section id=_Intel Customer Support>
+
+    <section id=_Intel_Customer_Support>
```
Comment (bapplejax) [inline 04LessonFour/in-class-project/index.html]: another id= case here

File: 04LessonFour/in-class-project/index.html [hunk 5]
```diff
@@ -2,64 +2,71 @@
...
-      <!--figure this shit out-->
-      <aside>
-        <h4> Architect of the Future</h4>
-        <p>
-          Intel strives to outdo current technology by supporting end-to-end solutions and designing system architecture that meets impending needs while exceeding current ones. See how innovation technologies such as Intel &reg; Solid-State Drives are leading to a future of enless possibilities.
-        </p>
-      </aside>
+    <section id=_SolidState>
+        <h3> Overview: Intel&reg; Solid-State Drives</h3>
+        <img src="img/ssd.jpg" />
```
Comment (bapplejax) [inline 04LessonFour/in-class-project/index.html:62]: and one on line 59 and 63.

File: 04LessonFour/in-class-project/index.html [hunk 6]
```diff
@@ -40,27 +40,27 @@
...
 
         <h3> Intel Customer Support</h3>
-        <p> Find Support Topics by Product, Drivers & Software, or ask the community <a id=customer_support href="www.http://www.intel.com/content/www/us/en/support/contact-support.html">read more</a>
+        <p> Find Support Topics by Product, Drivers & Software, or ask the community <a id="customer_support" href="www.http://www.intel.com/content/www/us/en/support/contact-support.html">read more</a>
 
 
         </p>
 
 
     </section>
```
Comment (bapplejax) [inline 04LessonFour/in-class-project/index.html:59]: great work! just be careful not to start any class names or IDs with special characters.

File: 07LessonSeven/homework/index.html [hunk 7]
```diff
@@ -1,10 +1,41 @@
...
-  <link rel="stylesheet" href="css/style.css">
+    <title></title>
+    <link rel="stylesheet" href="css/style.css">
 </head>
-<body>
 
+<body>
+    <header>
+        <h1>Profession Portal</h1>
+    </header>
```
Comment (bapplejax) [inline 07LessonSeven/homework/index.html:12]: this is looking very good! the only issue I have is that the heading, nav link, and green box don't exactly line up vertically. Try to make that space from the left the same amount.

File: 02LessonTwo/homework/Practice/Tina_Belcher_Resume.html [hunk 8]
```diff
@@ -9,7 +9,14 @@
...
+
+        <div class="burgers">
+<img  src="" alt="image"/>
+<img  src="" alt="image"/>
+<img
+ src="burgers.png" alt="image"/>
+</div>
+
         <p1>
             Small Town, New Jersey
```
Comment (bapplejax) [inline 02LessonTwo/homework/Practice/Tina_Belcher_Resume.html:21]: i love bob's burgers!

File: 07LessonSeven/homework/css/style.css [hunk 9]
```diff
@@ -26,7 +26,7 @@ margin-left: 10px;
   background-color: #55a747;
   float:left;
   width:35%;
-  margin-left:10%;
+  margin-left:125px;
   height: 175px;
```
Comment (bapplejax) [inline 07LessonSeven/homework/css/style.css:30]: great work making everything line up!

File: 08LessonEight/homework/exercise-06.html [hunk 10]
```diff
@@ -2,9 +2,40 @@
...
     <title>Exercise 06</title>
 </head>
 
-<body></body>
+<body>
+<div id="truck">
+  <img id="truck" src="img/truck.jpg" alt= truck />
+  <h1>SAFE & SECURE DELIVERY</h1>
+  <p>
+    Integer conue, elit non semper laoreet sed lectus orci posuh nisl tempor lacus felis ac mauris. Pellentesque in urna. Intege vitae felis vel magna posuere vestibulum.
```
Comment (bapplejax) [inline 08LessonEight/homework/exercise-06.html:14]: page looks perfect! great job on this one :dancer:

File: 08LessonEight/in-class-project/css/homework-style.css [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+    line-height: 20px;
+}
+p {
+    font-size: 11px;
+    line-height: 20px;
+    padding-left: 10px;
+    padding-top: 10px;
+}
+ul {
+    list-style-image: url(http://www.hrandtrainingdoctor.co.uk/wp-content/uploads/2013/07/Green-Tick-e1409164761628.jpg);
```
Comment (bapplejax) [inline 08LessonEight/in-class-project/css/homework-style.css:31]: way to find a way around the checkmark problem!

File: 08LessonEight/in-class-project/css/homework-style.css [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+    margin-top: auto;
+}
+h5 {
+    align-items: float: left;
+    padding-left: 20px;
+    font-size: 15px;
+    font-weight: lighter;
+    margin-top: auto;
+}
+h8 {
```
Comment (bapplejax) [inline 08LessonEight/in-class-project/css/homework-style.css:95]: h8?? so many levels!

File: 08LessonEight/in-class-project/css/homework-style.css [hunk 13]
```diff
@@ -0,0 +1,154 @@
...
+    padding-top: 20px;
+    padding-bottom: 140px;
+
+}
+#profession {
+    float: left;
+    border-right: #F7F7F7 solid;
+    width: 228px;
+    height: 120px;
+}
```
Comment (bapplejax) [inline 08LessonEight/in-class-project/css/homework-style.css:127]: these three all have the same stylings barring the border. consider using one class and one id to be more efficient

File: 08LessonEight/in-class-project/homework.html [hunk 14]
```diff
@@ -0,0 +1,114 @@
...
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla accumsan suscipit lacinia. Vivamus euismod varius odio eget.
+        </p>
+        <button type="button">Read More </button>
+
+    </section>
+
+    <aside>
+
+        <section class="product">
+            <img id="laptop" src="img/laptop.JPG" alt="laptop">
```
Comment (bapplejax) [inline 08LessonEight/in-class-project/homework.html:42]: this image and the next one have .jpg extensions. the first one was the only image with the capitalized extension. just go off whichever the exact file uses in the ATOM file tree

File: 09LessonNine/in-class-project/css/style.css [hunk 15]
```diff
@@ -0,0 +1,99 @@
+main {
+    text-align: center;
+    width: 100vw;
+    height: 100vh;
+    background: url(../img/1.jpg);
+    background-size: cover;
+}
+.opacity {
```
Comment (bapplejax) [inline 09LessonNine/in-class-project/css/style.css:8]: great use of the opacity! remember to utilize it on the services page as well though.

File: 09LessonNine/in-class-project/service.html [hunk 16]
```diff
@@ -0,0 +1,54 @@
...
+        <div class="landscaping">
+            <h1>LANDSCAPING & GARDENING</h1>
+            <h3>
+        We Do It For Your Sweet Homes And Working Places
+      </h3>
+            <p1>"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p1>
+
+        <nav>
+
+            <a id="more" href="#"> MORE ABOUT US </a>
```
Comment (bapplejax) [inline 09LessonNine/in-class-project/service.html:42]: i can't read these buttons! they're too small.

File: 10LessonTen/Checkpoint2/css/style.css [hunk 17]
```diff
@@ -0,0 +1,209 @@
...
+    background-color: rgba(251, 208, 192, 0.86);
+    padding: 20px;
+    width: 100%;
+}
+.opacity {
+    background-color: rgba(0, 0, 0, 0.3);
+    width: 100%;
+    color: white;
+    height: 90vh;
+}
```
Comment (bapplejax) [inline 10LessonTen/Checkpoint2/css/style.css:14]: I can side scroll when i view your site which means there's elements that add up to more than 100% width!

File: 10LessonTen/Checkpoint2/index.html [hunk 18]
```diff
@@ -1,10 +1,166 @@
...
+    <!--Fonts  -->
+    <link href='https://fonts.googleapis.com/css?family=Londrina+Outline' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Londrina+Solid' rel='stylesheet' type='text/css'>
+    <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro' rel='stylesheet' type='text/css'>
+
+    <!--Smooth Scroll Javascript -->
+    <script src="//ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
+
+    <script>
+        $(function() {
```
Comment (bapplejax) [inline 10LessonTen/Checkpoint2/index.html:17]: i like thissssss

File: 10LessonTen/Checkpoint2/index.html [hunk 19]
```diff
@@ -1,10 +1,166 @@
...
+            </br>
+            <img id="about_pic" src="http://www.newfoundreads.com/wp-content/uploads/2016/04/teacup-pig-1.jpg" />
+            <p>
+                Always fantasized of owning your very own teacup pig? Well you're in luck! PickAPig is an adoption agency here to help you pick the teacup pig of your dreams. Founded in 2009, PickAPig is the new modern way to get a pig in your home. <em> Friendly Warning: Teacup pigs stay teacup sized for a short period of time. </em> GET ONE TODAY!
+            </p>
+
+        </section>
+
+        </br>
+        </br>
```
Comment (bapplejax) [inline 10LessonTen/Checkpoint2/index.html]: lots of brs here! try just using margin or padding to create this space

File: 10LessonTen/Checkpoint2/index.html [hunk 20]
```diff
@@ -1,10 +1,166 @@
...
+                <h2>Because dogs are so last year</h2>
+                </br>
+                <a id="take" href="#adopt">Take One Home Today</a>
+            </div>
+        </header>
+        </br>
+        </br>
+        </br>
+        <section id="about">
+            <a href="#nav" class="back-to-top">Back to Top</a>
```
Comment (bapplejax) [inline 10LessonTen/Checkpoint2/index.html:70]: i love this appearing back to top button!! great use of javascript :dancers:
